### ISSUE TITLE
Issue running the script

### ISSUE THREAD

**Issue by fabston** [HilbertGilbertson/alfred-whencord#1]
Hi there,

I was trying to setup your workflow but unfortunately I am getting the following error message:

```
Code 127: /Users/fs/Library/Caches/com.runningwithcrayons.Alfred/Workflow Scripts/F2017730-4296-4048-A691-BF541040F1F5: line 2: /Users/fs/.bashrc: No such file or directory
/Users/fs/Library/Caches/com.runningwithcrayons.Alfred/Workflow Scripts/F2017730-4296-4048-A691-BF541040F1F5: line 3: php: command not found
```

I did install php as well as node using brew.

Appreciate your help. Thanks a lot!

**Comment by HilbertGilbertson** [comment]
Thanks for reporting this! I've now fixed this issue and released [v1.0.1](https://github.com/HilbertGilbertson/alfred-whencord/releases/tag/1.0.1).

I've now tested this on both Big Sur and a fresh install of Monterey just to be safe (darn MacOS changes make these things unnecessarily frustrating hehe), so all should now be working perfectly, but do feel free to let me know if otherwise.

**Comment by fabston** [comment]
Awesome, thanks a lot for the quick response and fix. `v1.0.1` did indeed fix it for me. Thanks again.

I am also using your other workflow, snowflake, which unfortunately is also throwing an error. To keep it clean I opened an issue on that [repo](https://github.com/HilbertGilbertson/alfred-snowcord/issues/1).
